jobs/JOBSPROC_DUMP_MONITOR

```diff
@@ -0,0 +1,277 @@
+  2) export PS4='+ ${SECONDS}s:L$LINENO + ';;         # add script line #
+  3) export PS4='+ ${SECONDS}s:${0##*/}:L$LINENO + ';;# add script name & line #
+  4) export PS4='+ ${0##*/}:L$LINENO + ';;  # script name & line (no seconds)
+  *) echo "Unrecognized DEBUG_LEVEL.  Stay with current PS4 setting";;
+esac
+
+###########################
+# Specify NET and RUN Name
+###########################
+export obsNET=${obsNET:-obsproc}
+export NET=${NET:-dump}
+export RUN=${RUN:-dump}
+
+set +x
+echo
+echo obsNET is ${obsNET:?"###FATAL ERROR \"obsNET\" is not set"}
+echo NET is ${NET:?"###FATAL ERROR \"NET\" is not set"}
+echo RUN is ${RUN:?"###FATAL ERROR \"RUN\" is not set"}
+echo
+set -x
+
+##############################
+# Specify cycle and time-mark
+##############################
+export cycle=t${cyc}z
+export tmmark=tm00
+
+###############################
+# Create new working directory
+###############################
+host=$(hostname -s)
+export DATA=${DATA:-$DATAROOT/${job}.${host}.$$}
+rm -rf $DATA
+mkdir -p $DATA
+cd $DATA
+if [[ "$RUN_ENVIR" != nco ]]; then
+  TMPDIR=$DATAROOT  # In the event TMPDIR is reset to a temporary LSF
+                    #  directory when batch jobs are submitted, set it to
+                    #  $DATAROOT so ensure it remains under the user's control
+                    #  (normally needed only if child script dumpjb executed)
+fi
+
+######################################
+# Set job log file for postmsg output
+######################################
+jlogfile=${jlogfile:-${DATA}/jlogfile}
+
+######################################
+# Determine Job Output Name on System
+######################################
+export pid=$$
+export outid="LL$job"
+[ $envir != prod ]  && export outid="LL${job}_${envir}"
+export jobid="${outid}.o${pid}"
+export pgmout="OUTPUT.${pid}"
+
+#################################################
+# SENDCOM      - Copy files to $COMOUT directory
+# SENDECF      - Flag Events on ECFLOW
+# SENDDBN      - Alert output file to TOC
+#################################################
+export SENDCOM=${SENDCOM:-YES}
+export SENDECF=${SENDECF:-YES}
+export SENDDBN=${SENDDBN:-YES}
+
+##########################
+# Specify Execution Areas
+##########################
+
+# -------------------------------
+# Paths for obsproc_network items
+# -------------------------------
+export HOMEobsproc=${HOMEobsproc:\
+-${PACKAGEROOT:?}/obsproc.${obsproc_ver:?\
+"###FATAL ERROR \"obsproc_ver\" is not set"}} #doesn't matter if HOMEobsproc set
+
+export SCRIPTSobsproc=${SCRIPTSobsproc:-$HOMEobsproc/scripts}
+export EXECobsproc=${EXECobsproc:-$HOMEobsproc/exec}
+export USHobsproc=${USHobsproc:-$HOMEobsproc/ush}
+
+# ---------------------------------------
+# Paths for dump and bufr_dumplist items
+# ---------------------------------------
+
+export HOMEbufr_dump=${HOMEbufr_dump:\
+-${PACKAGEROOT:?}/bufr_dump.${bufr_dump_ver:?\
+"###FATAL ERROR \"bufr_dump_ver\" is not set"}}
```

Comment:
These 3 lines are redundant and not required, since $HOMEbufr_dump is predefined when bufr_dump/1.0.0 module is loaded. I suggest removing them.